index.d.ts

```diff
@@ -10,6 +10,10 @@ declare namespace srcset {
 	}
 
 	interface SrcSetOptions {
+		/**
+		 * Enable or disable strict mode. 
```

Comment:
This is already clear from it being a boolean. Doc comments should explain the implications. What does strict mean. What happens when disabled. How does the behavior change.